lib/components/screens/lists/members/widgets/twitter_list_members.dart

```diff
@@ -29,7 +28,7 @@ class TwitterListMembers extends StatelessWidget {
             state.members,
             beginSlivers: [
               HarpySliverAppBar(
-                title: '${list.name} members',
+                title: '${name ?? ''} members',
```

Comment:
```suggestion
                title: name != null ? '$name members' : 'members',
```
Just a small thing to prevent the extra space when it has no name